app/src/main/java/com/keylesspalace/tusky/fragment/ViewImageFragment.kt

```diff
@@ -108,95 +111,129 @@ class ViewImageFragment : ViewMediaFragment() {
             }
         }
 
-        attacher = PhotoViewAttacher(binding.photoView).apply {
-            // This prevents conflicts with ViewPager
-            setAllowParentInterceptOnEdge(true)
+        val singleTapDetector = GestureDetectorCompat(
+            requireContext(),
+            object : GestureDetector.SimpleOnGestureListener() {
+                override fun onDown(e: MotionEvent) = true
+                override fun onSingleTapConfirmed(e: MotionEvent): Boolean {
+                    photoActionsListener.onPhotoTap()
+                    return false
+                }
+            }
+        )
 
-            // Clicking outside the photo closes the viewer.
-            setOnOutsidePhotoTapListener { photoActionsListener.onDismiss() }
-            setOnClickListener { onMediaTap() }
+        binding.photoView.setOnTouchCoordinatesListener(object : OnTouchCoordinatesListener {
+            var lastY: Float? = null
```

Comment:
`oneFingerY` ?

Because it serves two functions: track movement and detect change between one and two fingers.

Reply:
Went with `lastDragY`.